getting_started/introduction/godot_design_philosophy.rst

```diff
@@ -81,8 +81,8 @@ user experience. You can still work with external programs as long as
 there is an import plugin for it. Or you can create one, like the `Tiled
 Map Importer <https://github.com/vnen/godot-tiled-importer>`__.
 
-That is also partly why Godot offers its own programming languages
-GDScript and, along with C#. They're designed for the needs
+That is also partly why Godot offers its own programming language
+GDScript along with C#. They're designed for the needs
```

Comment:
This should also change to singular, it's only referring to GDScript.

C# is not designed for the needs of game developers and game designers, it's a general purpose language.

Reply:
I updated it to reference GDScript directly to be extra clear we're talking about it and not C#.
I also squashed everything down.